examples/getting-started/openapi/pkg/api/openapi_generated.go

```diff
@@ -0,0 +1,57 @@
+// +build !ignore_autogenerated
```

Comment:
Can the comment pass `hack/verify_boilerplate.py `?

Reply:
Yes